### ISSUE TITLE
App stays blank forever in macos 15

### ISSUE THREAD

**Issue by amit9838** [DevToys-app/DevToys#1279]
### Current behavior

App Stays blank on launch

### How to reproduce it (as minimally and precisely as possible)

Download the the app from homebew or install Devtoys.app
Launch the app
App stays blank forerver

### Expected behavior

Should show tools

### Screenshots

<img width="1435" alt="Screenshot 2024-06-30 at 10 42 41 AM" src="https://github.com/DevToys-app/DevToys/assets/61614402/8304f357-5e15-4e73-9819-3c61d319f5fc">

### Affected platforms

macOS

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

2.0 (latest)

### Relevant Assets/Logs

<img width="1052" alt="Screenshot 2024-06-30 at 10 50 16 AM" src="https://github.com/DevToys-app/DevToys/assets/61614402/a4a01979-391e-4504-bd0f-30cdacefbd92">

No logs in terminal ( I can't access settings)

**Comment by veler** [comment]
Hi,
Sorry for the inconvenience. Can you please share the logs available in `~/Library/Caches/com.devtoys/Logs/` ?
Thanks

**Comment by amit9838** [comment]
Sorry to say, but today morning I reverted to macos 14, Macos 15 was having some more issues other than this.

**Comment by IbrahimOmar91** [comment]
I have the same issue

Attached, logs files:
[log-20240630T012215.txt](https://github.com/user-attachments/files/16058710/log-20240630T012215.txt)
[log-20240630T015956.txt](https://github.com/user-attachments/files/16058711/log-20240630T015956.txt)
[log-20240630T022158.txt](https://github.com/user-attachments/files/16058712/log-20240630T022158.txt)
[log-20240701T232015.txt](https://github.com/user-attachments/files/16058713/log-20240701T232015.txt)

**Comment by veler** [comment]
Thanks! @IbrahimOmar91 , are you also on MacOS 15?

**Comment by IbrahimOmar91** [comment]
> Thanks! @IbrahimOmar91 , are you also on MacOS 15?

Yes

**Comment by veler** [comment]
Thanks everyone!
For context, DevToys 2.0 is a Blazor Hybrid app (see this as Electron but using .NET as backend). The implementation is forked from .NET MAUI. But it is NOT a .NET MAUI app.

Out of curiosity I tried out a .NET MAUI app on MacOS 15, and the issue reproduces too, so I opened an issue to MAUI. I will keep investigating on my side, but hopefully they can also help finding a fix, if I get stuck.

https://github.com/dotnet/maui/issues/23390
